tests/core/test_axes.py

```diff
@@ -45,7 +45,7 @@ def test_add_axis_with_group(simulation_builder, persons):
     simulation_builder.add_parallel_axis({'count': 2, 'name': 'salary', 'min': 0, 'max': 3000, 'period': '2018-11'})
     simulation_builder.expand_axes()
     assert simulation_builder.get_count('persons') == 4
-    assert simulation_builder.get_ids('persons') == ['Alicia0', 'Javier1', 'Alicia2', 'Javier3']
+    assert simulation_builder.get_ids('persons') == ['Alicia0', 'Javier0', 'Alicia1', 'Javier1']
```

Comment:
I'm not sure this is prudent; if two people have the same name then ids will be non-unique.

Reply:
That non-uniqueness would be consistent with the input data.
IMHO, the axe expand mecanism should not try to mitigate non-uniqueness issues.

Reply:
Regarding non-uniqueness, `SimulationBuilder` already ignores duplicated people _before_, as you can see with this test:

```python
def test_ignores_duplicated_people(simulation_builder, persons):
    simulation_builder.add_person_entity(persons, {'Alicia': {}, 'Alicia': {}, 'Alicia': {}})
    assert simulation_builder.get_count('persons') == 1
```

Reply:
After taking a look at the code, I guess this change won't have an impact on simulations as those ids are used solely in the context of axes expansion.